src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/NotificationService.java

```diff
@@ -79,35 +73,35 @@ public AlertStatus getAlerts() {
             alertStatus.setBackupRestoreRunningSince(backupRestoreStartedAt);
             alertStatus.setCurrentTime(OffsetDateTime.now(ZoneOffset.UTC));
         }
-        if (isSecurityServerFullyInitialized()) {
+
+        try {
             alertStatus.setGlobalConfValid(isGlobalConfValid());
-            alertStatus.setSoftTokenPinEntered(isSoftTokenPinEntered());
-        } else {
+            alertStatus.setGlobalConfValidCheckSuccess(true);
+        } catch (CodedException e) {
+            log.error("getting global conf status failed");
             alertStatus.setGlobalConfValid(false);
+            alertStatus.setGlobalConfValidCheckSuccess(false);
+        }
+
+        try {
+            alertStatus.setSoftTokenPinEntered(isSoftTokenPinEntered());
```

Comment:
Should we just `setSoftTokenPinEnteredCheckSuccess(false)` if we know that backup restore is running? Since doing `isSoftTokenPinEntered` will likely fail doing restore, and possibly cause the "could not list all tokens" error to appear with some delay?

Reply:
That makes sense. Both `soft_token_pin_entered` and `soft_token_pin_entered_check_success` are now set to `false` always when backup / restore is running, and the actual check is skipped.